client/src/components/EditProfile/EditForm/EditForm.tsx

```diff
@@ -80,17 +83,56 @@ const formValidation = Yup.object().shape({
 const EditForm = ({ handleSubmit }: Props): JSX.Element => {
   const classes = useStyles();
   const options = ['Male', 'Female', 'Other'];
+  const [isAvailable, setIsAvailable] = useState(false);
+
+  const daysOptions = ['Monday', 'Tuesday', 'Wednesday', 'Thrusday', 'Friday', 'Saturday', 'Sunday'];
+
+  const handleSwitch = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setIsAvailable(event.target.checked);
+  };
+
   return (
     <Box>
       <Formik initialValues={initialFormState} validationSchema={formValidation} onSubmit={handleSubmit}>
         {({ handleSubmit }) => (
           <Form onSubmit={handleSubmit}>
             <Grid container alignItems="center" spacing={2}>
               <Grid item xs={12}>
-                <Typography variant="h6" align="center">
+                <Typography variant="h5" align="center">
                   Edit Profile
                 </Typography>
               </Grid>
+              <Grid item xs={12} sm={3}>
+                <Typography variant="body1" color="textPrimary" className={classes.typography}>
+                  {isAvailable ? `I 'm available` : `I 'm not available`}
+                </Typography>
+              </Grid>
+              <Grid item xs={12} sm={8}>
+                <Switch color="error" onChange={handleSwitch} />
+              </Grid>
+              <Grid item xs={12} sm={3}>
+                <Typography variant="body1" color="textPrimary" className={classes.typography}>
+                  Availability
+                </Typography>
+              </Grid>
+              <Grid item xs={12} sm={8}>
+                <FormGroup>
+                  <Box display="flex" flexWrap="wrap">
+                    {daysOptions.map((day, i) => (
+                      <div key={i}>
+                        {isAvailable ? (
+                          <>
+                            <Field type="checkbox" name="availability" label={day} value={day} as={Checkbox} />
+                            <label>{day}</label>
+                          </>
+                        ) : (
+                          <FormControlLabel name="availability" control={<Checkbox disabled />} label={day} />
+                        )}
+                      </div>
+                    ))}
+                  </Box>
+                </FormGroup>
+              </Grid>
```

Comment:
Can we move this section of the UI into its own tab please 🙏🏼 We can keep the I'm available switch here though 👍🏼